### DISCUSSION TITLE
How should I call the new controlnet models?

### DISCUSSION THREAD

**Discussion by JasonS09** [Acly/krita-ai-diffusion#935] [Q&A]
I manually downloaded xinsir's scribble model for SDXL, but it doesn't seem to be working at all. I assume it's because the filename doesn't match to what the plugin is expecting. I dumped the workflow and it shows another one being selected. What should the filename be? Mine's currently controlnet-scribble-sdxl-1.0.safetensors.

**Comment by XylitolJ** [comment]
I also had the same problem, and this is how I handled it, hope it can help you (I'm not a coder)

Since because of these:

![image](https://github.com/Acly/krita-ai-diffusion/assets/16879990/5642abde-e27b-42cc-9263-c5f0297ceade)

and this:
![image](https://github.com/Acly/krita-ai-diffusion/assets/16879990/cdba7545-cad0-4822-82ee-cf48be4733e7)

**So I rename these Xinsir CN like this:**
![image](https://github.com/Acly/krita-ai-diffusion/assets/16879990/f10a7de6-2ffb-4377-8d4c-ff401e7e5e70)
 
Before adding "krita" to file name, it does not load xinsir model:
![image](https://github.com/Acly/krita-ai-diffusion/assets/16879990/c3e39096-7d16-43d1-9069-a522417c706a)

After adding "krita" to file name of xinsir CN models, re-run Krita, check client.log and saw this:
![image](https://github.com/Acly/krita-ai-diffusion/assets/16879990/0c55bf5b-0ecc-4af8-9244-b2f77a1592a3)

So, it work ~!

**Comment by Acly** [comment]
Basically it should work if the filepath matches `xinsirscribble` (partial match is ok, case-insensitive)

Seems reasonable to also match the hugging face repo name though (eg `scribble-sdxl`), already did that for the tile model.

I'm a bit surprised adding "krita" step was necessary for @XylitolJ - the xinsir models should be preferred even without it. I'll check later.
